src/azure-cli-core/azure/cli/core/commands/__init__.py

```diff
@@ -378,7 +378,7 @@ def get_command_table(module_name=None):
     If the module is not found, all commands are loaded.
     '''
     loaded = False
-    if module_name and module_name not in BLACKLISTED_MODS:
+    if module_name and module_name not in BLACKLISTED_MODS and module_name != 'sf':
```

Comment:
Add a TODO to remove this later.
Would you remove once the old module has been deprecated?